Makefile

```diff
@@ -12,38 +12,58 @@ base-image :
 base-notebook : base-image
 	cd base-notebook ; \
 	conda-lock lock --mamba -k explicit -f environment.yml -p linux-64; \
-	../generate-packages-list.py conda-linux-64.lock > packages.txt; \
+	../generate-packages-list.py conda-linux-64.lock > packages-64.txt; \
+	conda-lock lock --mamba -k explicit -f environment.yml -p linux-aarch64; \
+	../generate-packages-list.py conda-linux-aarch64.lock > packages-aarch64.txt; \
+	conda-lock lock --mamba -k explicit -f environment.yml -p linux-ppc64le; \
+	../generate-packages-list.py conda-linux-ppc64le.lock > packages-ppc64le.txt; \
```

Comment:
see comment above on unified lockfile. but i still really like a simple text file as well though! maybe we create a `artifacts` subdirectory to organize files not actually used by docker build?